resources/grafana/generated/dashboards/rhacs-central-dashboard.yaml

```diff
@@ -1,3 +1,4 @@
+# rhacs-central-dashboard.yaml is generated from rhacs-central.json
```

Comment:
You have to change the location in the `index.json` as well after moving the file.

Reply:
Thank you @stehessel, updated the `index.json`, can you explain the process on how these dashboards will be deployed onto clusters

Reply:
Sure, the observability operator (https://github.com/redhat-developer/observability-operator) has a pull mechanism which pulls the config from a configured git revision.